jobs/upload-dashboards-objects/templates/dashboards-objects/index-pattern/logs-app.json.erb

```diff
@@ -175,5 +175,7 @@ end
     "title": "<%= p('opensearch_config.app_index_prefix') %>*",
     "timeFieldName": "@timestamp",
     "fields": "<%= fields_json.to_json.gsub(/"/) { '\"' } %>"
-  }
+  },
+  "id": "<%= p('opensearch_config.app_index_prefix') %>*",
+  "type":"index-pattern"
```

Comment:
What do these lines do?

Reply:
This is the reason peters index was able to be there but not

Reply:
it also tells opensearch this is not a legacy object from elasticsearch